### DISCUSSION TITLE
Check the underlaying postgres schema created by edgdeb

### DISCUSSION THREAD

**Discussion by jordic** [geldata/gel#1813] [General]
I just started with the tutorial, and thaought it's awesome how you solve some 
of the common boring postgres parts, with a new and rich language.

Anyway, I'm just curious how do you manage behind it on the postgresql side. 

I checked docs and tutorials, but don't found how to connect to the internal postgresql instance, it's this possible?

**Comment by elprans** [comment]
If you run a developer build, you can use the `\psql` command in REPL.  Otherwise, you can use this command:

```
psql -U postgres -h${datadir} -p$(ps auxww | grep -e 'postgres.*.edgedb/\?\s*--port' | sed -r -e 's/.*--port=([[:digit:]]+).*/\1/g')
```

here `${datadir}` is the location of the EdgeDB data directory.

**Reply by jordic** [reply]
Thank you so much!
